woocommerce/woocommerce
# [e2e] External - Consolidate `playwright.config.js` for WPCOM

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Consolidate `playwright.config.js` for WPCOM. 
This one has left behind after all updates to e2e tests related to WPCOM env. 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. CI green - to be sure that changes in this PR are not affecting wp-env execution.
2. Locally - checkout this branch and run `pnpm test:e2e:with-env default-wpcom`. Make sure that all tests listed in `playwright.config.js` have started.
3. On-demand run [here](https://github.com/woocommerce/woocommerce/actions/runs/11146209122).



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Consolidate `playwright.config.js` for WPCOM

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
